### DISCUSSION TITLE
Apple Cinema Display - half as bright as it should be

### DISCUSSION THREAD

**Discussion by skaman82** [MonitorControl/MonitorControl#1381] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

My cinema display is about at 50% brightness on the max setting from the app on my M1 max MacBook (Ventura). I am only able to restore the full brightness by reconnecting the display to my old Mac pro and adjust it to correct brightness.

### Steps to reproduce

See above 

### Expected behavior

Display should reach its max brightness when the app slider is set to 100%

### Environment Information (please complete the following information)

```markdown
- macOS version: Ventura 
- Mac model: Mac book pro (M1 max)
- MonitorControl version: latest release version 
- Monitor(s): Apple Cinema Display 27"
- Apple Silicon/M1 (yes or no): yes
```

**Comment by Draeconis** [comment, marked as answer]
Under Preferences > Displays, enable 'Show advanced settings'.

Under your LED Cinema Display entry ensure that Brightness control is enabled, and set DDC max to '1023'.

**Reply by analogue-interface** [reply]
This works for me! Tahoe 26.4.  Apple Cinema Display.  MonCon 4.3.3

Thanks

**Comment by waydabber** [comment]
Exactly!

**Comment by NickelPhillips** [comment]
I had/have a very similar experience: When I installed MonitorControl, I watched the display automatically dim to about 50% brightness. Everything on MonitorControl and on MacOS Ventura is still set to 100%. _System Settings > Displays > Advanced_ offers no brightness control (see screenshot).  

On an M1 Mac mini + Apple LED Cinema Display (27-inch). Installed _MonitorControl.4.1.0.dmg_ 
![Screenshot 2023-05-31 at 3 56 20 PM](https://github.com/MonitorControl/MonitorControl/assets/135172712/b05e7dd0-8faa-4b29-b284-b329bc30995a)

**Comment by NickelPhillips** [comment]
Preferences > Displays > Advanced and setting DDC max to 255 worked! Should've tried that before posting.

**Reply by waydabber** [reply]
Yes, the standard solution is to set it to 255 but it seems like hardware revisions and firmware might differ. For some the 255 value does not seem to work, there are varying reports about this.

**Comment by turman11** [comment]
Hi there, for me worked to write DDC 255, but volume is still low, what figures must i write to gain the high volume? Thanks in advance.

**Reply by waydabber** [reply]
Hmm. I have no idea - wasn't aware that DDC volume control even works with the ACD. Doesn't it have USB audio?

**Comment by turman11** [comment]
Along with the brightness, the sound also decreased proportionally, DDC 255 worked on the brightness, but the sound remained the same.
I transmit both the image and the sound via Mini DisplayPort to USB-C. As for the MacBook, I have a Macbook Pro 14" M1 Pro
@waydabber

**Comment by waydabber** [comment]
Let's see, maybe other ACD users will chime in. I actually never had such a display so I don't have first hand experience.
